src/Microsoft.Framework.DependencyInjection/ActivatorUtilities.cs

```diff
@@ -143,8 +144,8 @@ private static MethodInfo GetMethodInfo<T>(Expression<T> expr)
                 }
                 else
                 {
-                    var parameterTypeExpression = Expression.Constant(parameterType);
-                    constructorArguments[i] = Expression.Call(serviceProvider, GetServiceInfo, parameterTypeExpression);
+                    var parameterTypeExpression = new Expression[] { serviceProvider, Expression.Constant(parameterType) };
```

Comment:
Since GetRequiredService is static method as opposed to GetService, I could not use the overload that was being used. I had to change the overload